lib/src/dependency.dart

```diff
@@ -139,7 +141,44 @@ class GitDependency extends Dependency {
   String get _info => 'url@$url';
 }
 
-Uri _parseUri(String value) => Uri.parse(value);
+Uri parseGitUri(String value) {
```

Comment:
[optional] could use `??` since we're focused on the 'what' of this value and not side effects.

```dart
Uri parseGitUri(String value) => _tryParseScpUri(value) ?? Uri.parse(value);
```

Reply:
ack